rebar.config

```diff
@@ -1,26 +1,59 @@
-    {erl_opts, [debug_info,
-                warn_missing_spec,
-                {i, ["include"]},
-                {src_dirs, ["src", "scenarios"]},
-                {parse_transform, lager_transform}]}.
+{erl_opts, [debug_info,
+            warn_missing_spec,
+            {i, ["include"]},
+            {src_dirs, ["src", "scenarios"]},
+            {parse_transform, lager_transform}]}.
 
-    {eunit_compile_opts, [nowarn_missing_spec]}.
+{eunit_compile_opts, [nowarn_missing_spec]}.
 
-    {require_otp_vsn, "^1[789]"}.
+{plugins, [pc,
+           rebar3_proper
+          ]}.
 
-    {deps, [
-            {lager, ".*", {git, "git://github.com/basho/lager.git", "3.2.1"}},
-            {jiffy, ".*", {git, "https://github.com/davisp/jiffy.git", "0.14.7"}},
-            {trails, ".*", {git, "https://github.com/inaka/cowboy-trails.git", {tag, "0.1.1"}}},
-            {cowboy_swagger, ".*", {git, "http://github.com/inaka/cowboy-swagger.git", {tag, "1.0.3"}}},
-            {escalus, ".*", {git, "git://github.com/esl/escalus.git", {tag, "3.0.0"}}},
-            {usec, ".*", {git, "git://github.com/esl/usec.git", {branch, "master"}}},
-            {exometer, ".*", {git, "git://github.com/Feuerlabs/exometer.git", "7a7bd8d2b52de4d90f65aa3f6044b0e988319b9e"}},
-            {lhttpc, ".*", {git, "git://github.com/esl/lhttpc.git", {branch, "otp-17-compat"}}},
-            {mochijson2, ".*", {git, "git://github.com/bjnortier/mochijson2.git", {branch, "master"}}},
-            {proper, ".*", {git, "git://github.com/manopapad/proper.git", {branch, "master"}}},
-            {recon, ".*", {git, "https://github.com/ferd/recon.git", "2.2.1"}},
-            {cowboy, ".*", {git, "https://github.com/ninenines/cowboy.git", "1.0.4"}},
-            {fusco, ".*", {git, "https://github.com/esl/fusco.git"}}
-           ]}.
+{require_otp_vsn, "^1[789]"}.
 
+{deps, [
+        {lager, "3.2.1"},
```

Comment:
I know this format is simpler and more readable. It happens form time to time that amoc is used a dep in project which is still compiled by rebar2. Could you get back to the longer dep spec compatible with rebar2?

Reply:
@michalslaski - end of 2017 now - and rebar (2) has been [deprecated](https://github.com/rebar/rebar/commit/a4d835bdea621f6d2ebc463f773fec620ee12581) since Apr 14, 2016.